heapster.go

```diff
@@ -35,8 +35,7 @@ import (
 )
 
 var (
-	argPollDuration    = flag.Duration("poll_duration", 10*time.Second, "The frequency at which heapster will poll for stats")
-	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats. Acceptable values are in the range [1 second, 'poll_duration')")
+	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats.")
```

Comment:
Got it. I agree with everything you mentioned! I intend to make the new stats APIs in kubelet support resolution. One option we have is to query more often, but ask for only the most recent stat. WDYT of that option?
With this change, my understanding is that heapster will drop all but one stat entry from every query. Have you tested that?